ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
+    background-color: rgb(255, 255, 255);
+    height: 330px;
+    width: 99%;
+    position: relative;
+    display: flex;
+}
+
+.carousel p {
+    margin: 4px 0px;
+}
+
+.carousel img {
+    background-color: rgb(236, 236, 236);
+    height: 70%;
+    width: 98%;
+    margin: 0px 10px;
+}
+
+
+.deal-discount p:first-child {
+    background-color: rgb(255, 0, 0);
+    color: rgb(255, 255, 255);
+    display: inline-block;
+    margin-left: 10px;
+}
+
+.deal-discount p:last-child {
+    color: red;
+    display: inline-block;
+}
+
+.euro-sign {
+    font-size: 0.5em;
+    position: relative;
```

Comment:
Only position absolute has top and right ... this is position relative, which makes me think they are useless

Reply:
yes, but in this case it should be position absolute. That way your euro sign will stay in the same place over and over again

Reply:
I Can't position this euro sign and cent, could you provide me a solution please?